specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerAppsPatches.json

```diff
@@ -0,0 +1,566 @@
+              "type": "string",
+              "description": "The framework and its version in the old run image for the target image."
+            },
+            "osAndVersion": {
+              "type": "string",
+              "description": "The OS name and its version in the old run image for the target image."
+            }
+          },
+          "description": "The old layer details in the target image.",
+          "readOnly": true,
+          "x-ms-client-flatten": true
+        },
+        "patchType": {
+          "type": "string",
+          "description": "The type for the patch.",
+          "enum": [
+            "FrameworkSecurity",
+            "OSSecurity",
+            "FrameworkAndOSSecurity",
+            "Other"
+          ],
+          "x-ms-enum": {
+            "name": "PatchType",
+            "modelAsString": true
+          },
+          "readOnly": true
+        }
+      },
+      "readOnly": true
+    },
+    "PatchProperties": {
+      "description": "Top level properties that describes current states of the patch resource",
+      "type": "object",
+      "properties": {
+        "targetEnvironmentId": {
+          "format": "arm-id",
+          "type": "string",
+          "description": "The id of the target environment for the patch."
+        },
+        "targetContainerAppId": {
+          "format": "arm-id",
+          "type": "string",
+          "description": "The id of the target container app for the patch."
+        },
+        "targetRevisionId": {
+          "format": "arm-id",
+          "type": "string",
+          "description": "The id of the target revision for the patch."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/PatchProvisioningState",
+          "description": "Patch provisioning state.",
+          "readOnly": true
+        },
+        "patchApplyStatus": {
+          "$ref": "#/definitions/PatchApplyStatus",
+          "description": "The status of the patch operation once it has been provisioned",
+          "readOnly": true
+        },
+        "createdTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC timestamp that describes when the patch object was created.",
+          "readOnly": true
+        },
+        "updatedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The UTC timestamp that describes when the patch object was last updated.",
+          "readOnly": true
+        },
+        "patchDetails": {
+          "description": "Detailed info describes the patch operation for the target container app.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PatchDetails"
+          },
+          "x-ms-identifiers": [],
+          "readOnly": true
+        }
+      },
+      "readOnly": true
+    },
+    "PatchCollection": {
+      "description": "Container App SourceToCloud patch collection",
```

Comment:
Could you please remove the `SourceToCloud` part? The fact that the patches are generated for the source to cloud scenario is not something that needs to be set in stone. If in a few months we had some patches generated for any given container app, we should still be able to reuse the same patch objects.

Reply:
removed in the latest commit.